bigquery_storage/tests/system/test_reader.py

```diff
@@ -15,9 +15,52 @@
 # limitations under the License.
 """System tests for reading rows from tables."""
 
+import copy
+import datetime as dt
+import decimal
+import json
+import io
+import re
+
 import pytest
+import pytz
 
+from google.cloud import bigquery
 from google.cloud import bigquery_storage_v1beta1
+from google.protobuf import timestamp_pb2
+
+
+def _add_rows(table_ref, new_data, partition_suffix=""):
```

Comment:
FWIW, we have an open feature request for such a method at https://github.com/googleapis/google-cloud-python/issues/4553

I propose we call that proposed method something along the line of `load_table_from_json` to be consistent with `load_table_from_dataframe`. Maybe we should implement that first so that we aren't duplicating code here?

Reply:
Depends on how fast we want this to be merged. If it can wait for a bit more, we can add that other method requested in #4553 first.

Reply:
If you'd rather merge this sooner rather than later, let's add a TODO with a link to that FR to remove the `_add_rows` function once #4553 is complete.

Reply:
Let's go with the TODO then, there's value in having these tests in place.

Also linked this thread from #4553 so that we don't forget about it.